frodo.js

```diff
@@ -1,42 +1,42 @@
 /**
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.